### PR TITLE
Assign different TEIDs to UL, DL traffic and verify TEIDs on pkts

### PR DESCRIPTION
**Pull request by Wailoks**
- Assign different TEIDs to UL and DL traffic in P4RT UP4 flows
- Verify the received TEIDs in `traffic.py` in all scenarios apart from the buffering ones, where we skip TEID verification

### CODE REVIEW

File: scenarios/bin/traffic.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 pkt_count = 0
 
 ue_addresses_expected = set()  # UE addresses that we expect to receive packets for/from
+downlink_teids_expected = set() # Downlink TEIDs that we expect to receive packets for
```
Comment (pierventre) [inline scenarios/bin/traffic.py:23]: we don't care when doing buffering. It is ok to count packets but we don't care if the TEIDs are 0 for some

**Review by daniele-moro — APPROVED** [review]

**Comment by Wailoks** [comment]
retest this please

**Comment by pierventre** [comment]
it looks like we are not in sync with the master

**Review by pierventre — APPROVED** [review]
Looks good please clarify my doubts and sync with the master

File: scenarios/pfcp-buffering.xml [hunk 2]
```diff
@@ -26,14 +26,15 @@
             <parallel var="${ENODEB#}">
                 <!-- Check Downlink only -->
                 <step name="Downlink-Pdn-Send-Udp-Buff-${#}"
-                      exec="mn-cmd pdn traffic.py send-udp -c 5 --flow-count 5 --ue-pool 17.0.${#-1}.0/24 --enb-addr 140.0.10${#-1}.1"/>
+                      exec="mn-cmd pdn traffic.py send-udp -c 16 --flow-count 5 --ue-pool 17.0.${#-1}.0/24 --enb-addr 140.0.10${#-1}.1"/>
```
[the remark this answers is not in the document]
Comment (Wailoks) [reply scenarios/pfcp-buffering.xml]: restored

**Pull request merged**